### PR TITLE
Use buttons on flow results to toggle all runs and responded runs

### PR DESCRIPTION
**Pull request by norkans7**
screencast 2019-01-23 15-35-57

### CODE REVIEW

**Comment by rowanseymour** [comment]
@norkans7 wouldn't it more consistent to use links as per https://github.com/nyaruka/rapidpro/pull/2242#issuecomment-456620049 ? Would match with the toggle controls on the archives page

**Comment by norkans7** [comment]
Sure, I did not know a place we use that so I tried the buttons

**Comment by norkans7** [comment]
screencast 2019-01-23 19-26-37

**Comment by ericnewcomer** [comment]
With this whole new row, we should just pull the "More Columns" drop down out of the header since it was only there to avoid adding a row at the top for controls. This would also avoid the jarring non-alignment between the form controls for searching / adding columns.

**Comment by norkans7** [comment]
results - sms form 2019-01-23 20-33-22

File: templates/flows/flow_results.haml [hunk 1]
```diff
@@ -584,37 +597,49 @@
...
-                %input.contact-search{type:"text", name:"q", placeholder:"Contact",
-                                      ic-get-from:"/flow/run_table/{{object.id}}/",
-                                      ic-on-beforeSend:'$("#results_run_table").empty()',
-                                      ic-include: "#only_responded",
-                                      ic-trigger-on:"keyup changed",
-                                      ic-trigger-delay:"500ms",
-                                      ic-target:"#results_run_table",
-                                      ic-indicator:"#indicator"}
+                .text
+                  Contact
```
Comment (rowanseymour) [inline templates/flows/flow_results.haml]: while you're in here may as well add trans tags, i.e. `-trans "Contact"`

**Comment by ericnewcomer** [comment]
Definitely looks better, change placeholder for search to either "Search Contacts" or just "Search

**Pull request merged**